pkg/configuration/parameters_test.go

```diff
@@ -0,0 +1,317 @@
+// See parameters.go for this package's comment
+package configuration
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"reflect"
+	"testing"
+)
+
+var (
+	emptyValue           = ""
+	fakeUrl              = "https://github.com/TurnsCoffeeIntoScripts/jira-api-resource"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [var fakeUrl should be fakeURL](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4777494)